PsfRuntime/Config.cpp

```diff
@@ -299,6 +299,12 @@ void load_json()
                 }
             }
         }
+        if (!file)
+        {
+            MessageBoxEx(NULL, L"�Config.json� was not found. Kindly add it under the package root folder and re-install the application.", L"MSIX Packet Support Framework", MB_OK | MB_ICONERROR, 0);

```

Comment:
Can we removed this unsupported chars from message?

Reply:
Done the change.